ee/api/test/test_conversation.py

```diff
@@ -623,3 +623,46 @@ def test_billing_context_validation_invalid_data(self):
                 call_args = mock_start_workflow_and_stream.call_args
                 workflow_inputs = call_args[0][1]
                 self.assertEqual(workflow_inputs.billing_context, None)
+
+    @patch("ee.api.conversation.is_team_limited")
+    def test_quota_limit_exceeded(self, mock_is_team_limited):
+        """Test that requests are blocked when team exceeds quota limits."""
+        mock_is_team_limited.return_value = True
+
+        response = self.client.post(
+            f"/api/environments/{self.team.id}/conversations/",
+            {
+                "content": "test query",
+                "trace_id": str(uuid.uuid4()),
+                "conversation": str(uuid.uuid4()),
+            },
+        )
+
+        self.assertEqual(response.status_code, status.HTTP_402_PAYMENT_REQUIRED)
+        self.assertEqual(
+            response.json()["detail"],
+            "Your organization reached its AI credit usage limit. Increase the limits in Billing settings, or ask an org admin to do so.",
+        )
+        mock_is_team_limited.assert_called_once()
+
+    @patch("ee.api.conversation.is_team_limited")
+    def test_quota_limit_not_exceeded(self, mock_is_team_limited):
+        """Test that requests proceed normally when team has not exceeded quota limits."""
+        mock_is_team_limited.return_value = False
+
+        with patch(
+            "ee.hogai.agent.executor.AgentExecutor.astream",
```

Comment:
@Twixes FYI this was failing after I merged master into the branch - this seems to be the fix - flagging just in case

```
FAILED ee/api/test/test_conversation.py::TestConversation::test_quota_limit_not_exceeded - AttributeError: module 'ee.hogai' has no attribute 'stream'
```